regulation-worker/internal/service/service.go

```diff
@@ -30,9 +30,11 @@ type JobSvc struct {
 	DestDetail destDetail
 }
 
-//called by looper
-//calls api-client.getJob(workspaceID)
-//calls api-client to get new job with workspaceID, which returns jobID.
+/* called by looper
+calls api-client.getJob(workspaceID)
+ calls api-client to get new job with workspaceID, which returns jobID.
+ */
+
```

Comment:
[Minor]
Line comments are the norm; block comments appear mostly as package comments. ([Ref](https://go.dev/doc/effective_go#commentary))